source/jsonnet/england-wales/individual/blocks/employment/mainly_work_in_uk.jsonnet

```diff
@@ -0,0 +1,121 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local question(title) = {
+  id: 'mainly-work-in-uk-question',
+  title: title,
+  type: 'General',
+  answers: [
+    {
+      id: 'mainly-work-in-uk-answer',
+      mandatory: false,
+      options: [
+        {
+          label: 'Yes',
+          value: 'Yes',
+        },
+        {
+          label: 'No',
+          value: 'No',
+        },
+      ],
+      type: 'Radio',
+    },
+  ],
+};
+
+local nonProxyTitle = 'Do you mainly work in the UK?';
+local proxyTitle = {
+  text: 'Does <em>{person_name}</em> mainly work in the UK?',
+  placeholders: [
+    placeholders.personName,
+  ],
+};
+
+{
+  type: 'Question',
+  id: 'mainly-work-in-uk',
+  question_variants: [
+    {
+      question: question(nonProxyTitle),
+      when: [rules.isNotProxy],
+    },
+    {
+      question: question(proxyTitle),
+      when: [rules.isProxy],
+    },
+  ],
+  routing_rules: [
+    {
+      goto: {
+        block: 'employer-address-workplace',
+        when: [
+          {
+            id: 'employer-type-of-address-answer',
+            condition: 'equals',
+            value: 'At a workplace',
+          },
+          {
+            id: 'mainly-work-in-uk-answer',
+            condition: 'equals',
+            value: 'Yes',
```

Comment:
Instead of repeating this rule for `not set`, if you did `not equals` `No`, you should get the same behaviour if I am not mistaken?.

Same for the address-depot routing.